### PR TITLE
NEXT-00000 - Changed typo in image gallery config

### PR DESCRIPTION
**Pull request by PheysX**
### 2. What does this change do, exactly?
Changed `enitiy` to `entity`

### 5. Checklist

- [x] I have rebased my changes to remove merge conflicts
- [ ] I have written tests and verified that they fail without my change
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-implement-new-changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### <samp>🤖 Generated by Copilot at 38f08b7</samp>

This pull request fixes a typo in the `entity` property of the image gallery element config in the administration module. This typo prevented the selection of media items for the image gallery element in the CMS. The fix affects the JavaScript and Twig files of the element config and adds a changelog entry.

### <samp>🤖 Generated by Copilot at 38f08b7</samp>

* Fix typo in `entity` property of image gallery element config ([link](https://github.com/shopware/shopware/pull/3364/files?diff=unified&w=0#diff-3b8c60d49809fb5621c1b89916f24eb16c00760509c899e8d08a57a045fe647dL25-R25), [link](https://github.com/shopware/shopware/pull/3364/files?diff=unified&w=0#diff-1f11ed50c0dc9eca6f7228bd20d9b86d54519837dcb04adbfd84d356f511d7e1L61-R61))
* Add changelog entry for the typo fix ([link](https://github.com/shopware/shopware/pull/3364/files?diff=unified&w=0#diff-ba7aa960c7acfb00a08b4a306569cfc68b6c96de5aed55f3f4cbd91bfc2c38e1R1-R11))

### CODE REVIEW

File: src/Administration/Resources/app/administration/src/module/sw-cms/elements/image-gallery/config/index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default {
         return {
             mediaModalIsOpen: false,
             initialFolderId: null,
-            enitiy: this.element,
+            entity: this.element,
```
Comment (seggewiss) [inline src/Administration/Resources/app/administration/src/module/sw-cms/elements/image-gallery/config/index.js:27]: This is a breaking change. The typo property needs to stay and you need to add a comment above
/* @deprecated tag:v6.6.0 - Will be removed use entity instead. */

Comment (PheysX) [reply src/Administration/Resources/app/administration/src/module/sw-cms/elements/image-gallery/config/index.js:27]: Updated it and change changelog to "Deprecated"

**Comment by PheysX** [comment]
Hey @seggewiss, is everything fine now? 😄

**Comment by seggewiss** [comment]
@PheysX Sorry for the late response importing the change now 👍

**Comment by PheysX** [comment]
Thx, can I also get a hacktoberfest label? :D @seggewiss

**Pull request closed without merging**